geode-management/src/main/java/org/apache/geode/management/configuration/ClassName.java

```diff
@@ -134,4 +135,12 @@ public boolean equals(Object o) {
         && this.getInitProperties().equals(that.getInitProperties());
   }
 
+  public static boolean isClassNameValid(String className) {
```

Comment:
I'd prefer that this method be on an internal class (like the ManagementHelper class you removed) instead of exposing it to users on the external api.

Reply:
I don't see the harm of exposing it to users. I really don't like the general purpose XXXHelper class to hold these methods. It makes it hard to find where it is